pkg/katautils/config-settings.go

```diff
@@ -16,6 +16,7 @@ var defaultKernelPath = "/usr/share/kata-containers/vmlinuz.container"
 var defaultInitrdPath = "/usr/share/kata-containers/kata-containers-initrd.img"
 var defaultFirmwarePath = ""
 var defaultMachineAccelerators = ""
+var defaultCPUFeatures = ""
```

Comment:
afaik, this is for shimv2, should we set this to `pmu=off` by default?  otherwise shimv2 CI won't pass .. ?

cc @lifupan

Reply:
Actually this file would be imported by both of cli and shimv2, and it would be override  by the setting from runtime configuration file.